codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttConnectPayload.java

```diff
@@ -51,6 +66,10 @@ public String willTopic() {
     }
 
     public String willMessage() {
```

Comment:
Please deprecate this method and refer to the `willMessageInBytes` in the `@deprecated` javadocs.